### ISSUE TITLE
Too wordy, it should be just successful return code 200

### ISSUE THREAD

**Issue by vietlq** [jpmorganchase/quorum-examples#146]
https://github.com/jpmorganchase/quorum-examples/blob/b7d49774bdedeb27d5084c0e9b8b5acc83b57701/examples/7nodes/tessera-start.sh#L125

Use command like this:

```
if [ $(curl -o -I -L -s -w "%{http_code}" yahoo.com) -eq 200 ]; then echo good; else echo bad; fi
```

See https://stackoverflow.com/a/46005602/128028

Pre-condition: The `upcheck` from Tessera must return HTTP code 200 only on success & ready state. Anything else must return != 200.

**Comment by namtruong** [comment]
@vietlq thanks for the suggestion - however I think I would prefer the API endpoint result to be explicit. Clients could've made the call to the wrong API and still get a 200

**Comment by vietlq** [comment]
Then the test should be all CAPS and no spaces, just underscores for max readability

**Comment by Krish1979** [comment]
thanks for the suggestion however we are happy with the response we provide for uncheck.

**Comment by vietlq** [comment]
@Krish1979 you can't even search for `"I'm up!"` (with or without double quotes) via GitHub. That's a bad design pattern. If you used `TM_READY` then yes, it's totally searchable and has limit char set.

https://github.com/jpmorganchase/tessera/search?q=%22I%27m+up%21%22&unscoped_q=%22I%27m+up%21%22

https://github.com/jpmorganchase/quorum-examples/search?q=%22I%27m+up%21%22&unscoped_q=%22I%27m+up%21%22

**Comment by vietlq** [comment]
The string `"I'm up!"` also overcomplicates bash script:

Without `set -e`:

```
node5_1       | sh: up!: unknown operand
```

With `set -e`:

```
 set -e 
 if [ ! "${result}" == "I'm up!" ]; then 
     echo "Node ${i} is not yet listening on http" 
     DOWN=true 
 fi
```
